libs/journeys/ui/src/components/Video/VideoStats/VideoStats.tsx

```
@@ -7,107 +7,104 @@ import { useCallback, useEffect, useState } from 'react'
+      endAt
+    )
+
+    // Update all stats at once
+    setStats({
+      currentTime: adjustedCurrentTime,
+      duration,
+      buffered: formattedBuffered,
+      seekable: formattedSeekable,
+      currentQuality: getCurrentQuality({ player, t })
+    })
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)